packages/components/src/components/Button/index.tsx

```diff
@@ -46,6 +46,19 @@ const variantStyles = {
       color: 'foreground',
     },
   },
+  ghost: {
+    backgroundColor: 'transparent',
+    color: 'mutedForeground',
+    // same technique as primary
+    ':hover:not(:disabled)': {
+      color: 'foreground',
+      backgroundColor: '#E5E5E51A',
+    },
+    ':focus:not(:disabled)': {
+      color: 'foreground',
+      backgroundColor: '#E5E5E51A',
+    },
```

Comment:
Does the following work?

```suggestion
    ':hover:not(:disabled), :focus:not(:disabled)'': {
      color: 'foreground',
      backgroundColor: '#E5E5E51A',
    },
```